### ISSUE TITLE
Unable to open Makefile in GoLand 2020.1.1 with Makefile plugin enabled

### ISSUE THREAD

**Issue by aravindhp** [kropp/intellij-makefile#162]
If I have the Makefile plugin enabled I am unable to open this [Makefile](https://github.com/openshift/windows-machine-config-bootstrapper/blob/master/Makefile) in GoLand 2020.1.1 running on Fedora 31.  The IDE gets stuck at "Loading". If I disable the plugin I am able to load the file. Is there a fix or workaround for this?

**Comment by singalen** [comment]
I believe idea.log would be helpful. https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files

**Comment by ichekrygin** [comment]
Possibly related:
```
java.lang.AssertionError: (393,393), text: ifndef, language = Language: Makefile
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:180)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:104)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:85)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:70)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:57)
	at name.kropp.intellij.makefile.MakefileFoldingBuilder$MakefileConditionalFoldingDescriptor.<init>(MakefileFoldingBuilder.kt:58)
	at name.kropp.intellij.makefile.MakefileFoldingBuilder.buildFoldRegions(MakefileFoldingBuilder.kt:24)
	at com.intellij.lang.folding.LanguageFolding.buildFoldingDescriptorsNoPlaceholderCaching(LanguageFolding.java:92)
	at com.intellij.lang.folding.LanguageFolding.buildFoldingDescriptors(LanguageFolding.java:75)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.getFoldingsFor(FoldingUpdate.java:243)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.getFoldingsFor(FoldingUpdate.java:223)
	at com.intellij.codeInsight.folding.impl.CodeFoldingManagerImpl.buildInitialFoldings(CodeFoldingManagerImpl.java:140)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl.loadEditorInBackground(PsiAwareTextEditorImpl.java:54)
	at com.intellij.openapi.fileEditor.impl.text.AsyncEditorLoader.lambda$null$0(AsyncEditorLoader.java:93)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:739)
	at com.intellij.openapi.fileEditor.impl.text.AsyncEditorLoader.lambda$scheduleLoading$1(AsyncEditorLoader.java:91)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.insideReadAction(NonBlockingReadActionImpl.java:485)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$attemptComputation$2(NonBlockingReadActionImpl.java:442)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:75)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:129)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:625)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:570)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:61)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:118)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:75)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.attemptComputation(NonBlockingReadActionImpl.java:450)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$0(NonBlockingReadActionImpl.java:383)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:210)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:190)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Plugin version: 3.0.1
GoLand info: GoLand 2020.1
Build #GO-201.6668.125, built on April 8, 2020
Licensed to Illya Chekrygin
Subscription is active until September 13, 2020
Runtime version: 11.0.6+8-b765.25 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 16
Registry: run.processes.with.pty=TRUE, ide.completion.variant.limit=500, suggest.all.run.configurations.from.context=TRUE, ideFeaturesTrainer.welcomeScreen.tutorialsTree=TRUE
Non-Bundled Plugins: com.databricks, name.kropp.intellij.makefile

**Comment by aravindhp** [comment]
@singalen [idea.log](https://github.com/kropp/intellij-makefile/files/4565791/idea.log)
LMK if you need anything else.

**Comment by kropp** [comment]
Fixed in 3.1
